include/llbuild/BuildSystem/BuildSystemFrontend.h

```diff
@@ -116,7 +116,7 @@ class BuildSystemFrontendDelegate : public BuildSystemDelegate {
   /// Handle used to communicate information about a launched process.
   struct ProcessHandle {
     /// Opaque ID.
-    uintptr_t id;
+    uint64_t id;
```

Comment:
I am personally fine with this, as I’m pretty sure that is what I originally used.  However, I think this was changed alongside some of the efforts to build out Windows compatibility.  We should be sure that this doesn’t break that.  cc @compnerd